test/setup-fedora-govm.sh

```diff
@@ -80,9 +80,9 @@ while ! dnf install -y $packages; do
     fi
     cnt=$(($cnt + 1))
     # If it works, proceed immediately. If it fails, sleep and try again without aborting on an error.
-    if ! dnf update --refresh; then
+    if ! dnf update -y --refresh; then
```

Comment:
Could the "aborting on an error" mentioned in above comment have been caused by lack of `-y` option ?

Reply:
No. I think there were simply some intermittent network issues.